modules/administration-guide/partials/assembly_uninstalling-the-che-instance-and-the-che-operator.adoc

```diff
@@ -1,7 +1,52 @@
 [id="uninstalling-the-che-instance-and-the-che-operator_{context}"]
 = Uninstalling the {prod-short} instance and the {prod} Operator
 
-After uninstalling the {devworkspace} Operator, uninstall the {prod-short} instance and the {prod} Operator by using either the OpenShift web console or the OpenShift CLI.
+After uninstalling the {devworkspace} Operator, uninstall the {prod-short} instance and the {prod} Operator.
 
-include::partial$proc_uninstalling-che-by-using-the-openshift-web-console.adoc[leveloffset=0]
-include::partial$proc_uninstalling-che-by-using-the-openshift-cli.adoc[leveloffset=0]
\ No newline at end of file
+.Prerequisites
+
+* OperatorHub was used to install the {prod-short} instance on {ocp} or {osd}.
+
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+
+
+.Procedure
```

Comment:
This procedure should be reworked to mention the single command to uninstall the {prod-short}: 
`{prod-cli} server:delete`

Reply:
Nice catch. That calls for a new task with proper testing.

Reply:
@tolusha meaning we can drop this procedure and keep only the other uninstall procedure? That would be great!

Reply:
Yes, we can.
I tested `server:delete` and it works perfectly.

Reply:
Done. Uninstalling Che will look much simpler.